keyboards/keebio/iris/readme.md

```diff
@@ -15,6 +15,10 @@ Example of flashing this keyboard:
 
     make keebio/iris/rev2:default:avrdude
 
-See [build environment setup](https://docs.qmk.fm/#/newbs_getting_started) then the [make instructions](https://docs.qmk.fm/#/getting_started_make_guide) for more information.
+Note that Iris rev3 uses the `dfu` bootloader:
+
+    make keebio/iris/rev3:default:dfu
+
+See [build environment setup](https://docs.qmk.fm/#/newbs_getting_started) then the [make instructions](https://docs.qmk.fm/#/getting_started_make_guide) for more information. Linux users should consult options for [`atmel-dfu` `udev` rules](https://docs.qmk.fm/#/faq_build?id=linux-udev-rules).
```

Comment:
While the first part is fine, I am not sure that this should be here.  Specifically, this info is not board specific, and applies to any board.  As such, I feel that it's outside of the scope for what this readme is supposed to be.

```suggestion
```